src/com/massivecraft/factions/cmd/CmdFactionsRelationCheck.java

```diff
@@ -0,0 +1,38 @@
+package com.massivecraft.factions.cmd;
+
+import com.massivecraft.factions.Rel;
+import com.massivecraft.factions.cmd.type.TypeFaction;
+import com.massivecraft.factions.entity.Faction;
+import com.massivecraft.massivecore.MassiveException;
+
+public class CmdFactionsRelationCheck extends FactionsCommand
+{
+	// -------------------------------------------- //
+	// CONSTRUCT
+	// -------------------------------------------- //
+
+	public CmdFactionsRelationCheck()
+	{
+		// Parameter
+		this.addParameter(TypeFaction.get(), "faction");
+		this.addParameter(TypeFaction.get(), "faction", "you");
+	}
+
+	// -------------------------------------------- //
+	// OVERRIDE
+	// -------------------------------------------- //
+	
+	@Override
+	public void perform() throws MassiveException
+	{
+		// Args
+		final Faction faction1 = this.readArg();
+		final Faction faction2 = this.readArg(msenderFaction);
+		
+		Rel relationBetweenFactions = faction1.getRelationTo(faction2);
+		String relationText = relationBetweenFactions.getColor().toString() + relationBetweenFactions.getDescFactionMany();
+		if (relationBetweenFactions == Rel.MEMBER && msenderFaction != faction1) relationText = "<silver>the same faction";
+		
+		msg(faction1.getName(msenderFaction) +"<i> and "+ faction2.getName(msenderFaction) +"<i> are "+ relationText + "<i>.");
```

Comment:
I think this would be cleaner using the format syntax rather than concatenation. There is a var-arg version of the msg method which ends up processing the content using the parse command found here  https://github.com/MassiveCraft/MassiveCore/blob/master/src/com/massivecraft/massivecore/util/Txt.java#L184

Reply:
Updated to make use of parse functionality.